packages/core/utils/src/geometry.ts

```diff
@@ -1,64 +0,0 @@
-const SIDE_OPTIONS = ['top', 'right', 'bottom', 'left'] as const;
-const ALIGN_OPTIONS = ['start', 'center', 'end'] as const;
-
-type Axis = 'x' | 'y';
-
-type Side = typeof SIDE_OPTIONS[number];
-
-type Align = typeof ALIGN_OPTIONS[number];
-
-type Point = { x: number; y: number };
-
-type Size = { width: number; height: number };
-
-/**
- * Creates a rect (`ClientRect`) based on a Size and and a position (x, y).
- * This is useful to compute the rect of an element without having to actually move it.
- */
-function makeRect({ width, height }: Size, { x, y }: Point): ClientRect {
```

Comment:
Looking at this function, it was the only one reused so we said we'd move it into `@radix-ui/rect` but then I reconsidered given it's just using `DOMRect.fromRect`. I thought back then this API I exposed was nicer, but with spreading it's just as convenient, so you'll see I've updated all use cases accordingly.